lib/taskana-simplehistory-provider/src/main/java/pro/taskana/simplehistory/impl/HistoryQueryImpl.java

```diff
@@ -0,0 +1,336 @@
+package pro.taskana.simplehistory.impl;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import pro.taskana.ClassificationQuery;
+import pro.taskana.TimeInterval;
+import pro.taskana.BaseQuery.SortDirection;
+import pro.taskana.history.api.TaskHistoryEvent;
+import pro.taskana.simplehistory.impl.mappings.HistoryQueryMapper;
+import pro.taskana.simplehistory.query.HistoryQuery;
+import pro.taskana.simplehistory.query.HistoryQueryColumnName;
+
+public class HistoryQueryImpl implements HistoryQuery {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(HistoryQueryImpl.class);
+
+    private TaskanaHistoryEngineImpl taskanaHistoryEngine;
+    private HistoryQueryMapper historyQueryMapper;
+
+    private HistoryQueryColumnName columnName;
+    private List<String> orderBy;
+    private List<String> orderColumns;
+    private int max_rows;  // limit for rows. used to make list(offset, limit) and single() more efficient.
+
+    private String[] businessProcessIdIn;
+    private String[] parentBusinessProcessIdIn;
+    private String[] taskIdIn;
+    private String[] typeIn;
```

Comment:
could you change the property's name to "eventType" and its methods' name? It's some confusing